theconfluenceBOT/new.py

```diff
@@ -1,13 +1,22 @@
+#!/usr/bin/python3
+import sys
 import praw
-from praw.models import MoreComments
-from prawcore.exceptions import Forbidden
-
 import random
 import time
 import math
+import logging
+from praw.models import MoreComments
+from praw.exceptions import RedditAPIException
 from datetime import datetime
 from os import environ as env
 
+LOG_FORMAT = "%(levelname)-8s |%(asctime)-25s| %(name)-45s | %(funcName)-25s at line %(lineno)-5s:%(message)s\n"
+logging.basicConfig(
+    filename="/dev/stdout", filemode="w", level=logging.INFO, format=LOG_FORMAT
```

Comment:
This is really nice, but this is too cluttered - seems like Heroku already provides some of the same logging format
![image](https://user-images.githubusercontent.com/36007346/151671012-8fae43ce-f4f8-4277-bcc6-86fffe4c9587.png)

Reply:
Also - I don't think this is supported on Windows. Not really an issue but might be good to add for local testing.